bin/configure.sh

```
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Configure script used by fluo-dev & fluo-deploy
+
+BIN_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+PC_HOME=$( cd "$( dirname "$BIN_DIR" )" && pwd )
+
+if [ "$#" -ne 1 ]; then
```

I put the check in both files because I wanted the run command to fail fast